<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phần mềm khí tượng thủy văn</string>
  </property>
  <property name="locale">
   <locale language="Vietnamese" country="Vietnam"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="showFolder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="flow">
         <enum>QListView::TopToBottom</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>false</bool>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="gridSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="prev">
          <property name="text">
           <string>Trước</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next">
          <property name="text">
           <string>Tiếp</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="MyGraphicsView" name="Mask">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFolder"/>
   <addaction name="separator"/>
   <addaction name="actionZoom"/>
   <addaction name="actionpdf2png"/>
   <addaction name="actionDrawingLine"/>
  </widget>
  <action name="actionOpenFolder">
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/icon/icons/openFolder.png</normaloff>:/icon/icons/openFolder.png</iconset>
   </property>
   <property name="text">
    <string>Mở Thư Mục</string>
   </property>
   <property name="toolTip">
    <string>Mở Thư Mục</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/icon/icons/zoomOff.png</normaloff>
     <normalon>:/icon/icons/zoomOn.png</normalon>:/icon/icons/zoomOff.png</iconset>
   </property>
   <property name="text">
    <string>Thu phóng</string>
   </property>
   <property name="toolTip">
    <string>Phóng to, thu nhỏ bằng con lăn chuột</string>
   </property>
  </action>
  <action name="actionpdf2png">
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/icon/icons/splitImg.png</normaloff>:/icon/icons/splitImg.png</iconset>
   </property>
   <property name="text">
    <string>Tách đồ thị</string>
   </property>
  </action>
  <action name="actionDrawingLine">
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/icon/icons/drawLine.png</normaloff>:/icon/icons/drawLine.png</iconset>
   </property>
   <property name="text">
    <string>Vẽ đường đồ thị</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
